content/concepts/did-ddo.md

```diff
@@ -34,98 +43,120 @@ In Ocean, a DID is a string that looks like:
+| **`author`**        | Text                  |**Yes**   | Name of the entity generating this data (e.g. Tfl, Disney Corp, etc.).                                                                                                                            |
+| **`license`**       | Text                  |**Yes**   | Short name referencing the license of the asset (e.g. Public Domain, CC-0, CC-BY, No License Specified, etc. ). If it's not specified, the following value will be added: "No License Specified". |
+| **`links`**           | Array of Link | No       | Mapping of links for data samples, or links to find out more information. Links may be to either a URL or another Asset. We expect marketplaces to converge on agreements of typical formats for linked data: The Ocean Protocol itself does not mandate any specific formats as these requirements are likely to be domain-specific. The links array can be an empty array, but if there is a link object in it, then an "url" is required in that link object. |
+| **`inLanguage`**      | Text          | No       | The language of the content. Please use one of the language codes from the [IETF BCP 47 standard](https://tools.ietf.org/html/bcp47)|
+| **`categories`**      | Array of Text | No       | Optional array of categories associated to the asset. Note: recommended to use `"tags"` instead of this.   |
+| **`tags`**            | Array of Text | No       | Array of keywords or tags used to describe this content. Empty by default. |
+| **`additionalInformation`**            | Object | No       | Stores additional information, this is customizable by publisher |
 
-- `metadata` - describing the asset
-- `access` - describing how the asset can be downloaded
-- `compute` - describing how the asset can be computed upon
+Depending on the asset type (dataset, algorithm), there are different metadata attributes supported:
 
-Each asset has a `metadata` service and at least one other service.
+### Algorithm attributes
 
-Each service is distinguished by the `DDO.service.type` attribute.
+An asset of type `algorithm` has the following additional attributes under `algorithm` in metadata object:
 
-Each service has an `attributes` section holding the information related to the service. That section _must_ have a `main` sub-section, holding all the mandatory information that a service has to provide.
+| Attribute           |   Type                |   Required  | Description                                         |
+| ------------------- | ----------------------| ----------- |--------------------------------------------------- |
+| **`language`**      | `string`              | no          | Language used to implement the software |
+| **`format`**        | `string`              | no          | Packaging format of the software. |
```

Comment:
can you share more information, give some examples

Reply:
removed that, we are not using it